client/src/components/Feed/StyledComment.js

```diff
@@ -16,7 +16,7 @@ const StyledComment = styled(Comment)`
     .ant-comment-inner {
       padding: 1rem 0;
       .ant-comment-content {
-        max-width: 30rem;
+        width: 45rem;
```

Comment:
This seems to be causing issues with narrower widths:

![image](https://user-images.githubusercontent.com/10546720/86790983-081bdf00-c037-11ea-9498-fe7ade74c2d9.png)

Maybe if post page area wasn't so narrow that would be okay but it doesn't solve it for mobile:

![image](https://user-images.githubusercontent.com/10546720/86791057-1ff36300-c037-11ea-9db0-900af4be86ef.png)


Can we break we let it break onto multiple lines if necessary?

Reply:
ah thanks! great catch!